scripts/validate-dist/source-validate-types.ts

```diff
@@ -8,7 +8,6 @@ export type {
 	FontWeightDefinition,
 	Option,
 	TypographyStyles,
-	TypographySizes,
```

Comment:
Since this is no longer being removed, can we reinstate this type here? Then if we regenerate the changeset, it should remove the bump to `validate-dist`